declare var test on global scope issue

Great to see TypeScript definition files with the package.
Unfortunately, we're having issues in a project that also uses Mocha. Both libraries have a `test` declaration on global scope, so TypeScript is throwing the following error:

```
Error TS2403: Subsequent variable declarations must have the same type.  Variable 'test' must be of type 'TestFn', but here has type 'ITestDefinition'.
```

In `testcafe` package:
`declare var test: TestFn;`

In `@types/mocha`
`declare var test: Mocha.ITestDefinition;`

It's global scope all over again.
I would prefer to import `test` from the Testcafe package. Or is there some other solution to this?

I still have to check our situation, I am now on holiday 🏖 , maybe @peterwestendorp can check, but when – in your example – I add the `e2e` directory to the included sources, then it will not compile.

```json
{
    "include": [
        "unit/**/*.ts",
        "e2e/**/*.ts"
    ]
}
```

```
$ npx tsc
e2e/test.ts(1,26): error TS2307: Cannot find module 'testcafe'.
e2e/test.ts(3,1): error TS2304: Cannot find name 'fixture'.
e2e/test.ts(7,13): error TS2339: Property 'wait' does not exist on type 'MochaDone'.
```

So it is hard to compare, because it might also be the case that the definition file is not found anyway, if so changing `"types"` will not affect anything.
In the last error, it uses the Mocha definitions. But this might be related to something else, it is hard to debug these definition things...
Changing the version to `0.16.2` has the seem results.

When the `e2e` directory is excluded in the `tsconfig.json`, I also notice some editors have problems when figuring out how to deal with the TypeScript.

Anyway, it doesn't seem that these issues are related to the original issue, where the compilation in TestCafe was causing the mocha definition conflict...

So, back to the _original_ case, running the TypeScript tests with TestCafe, when I add this to the `package.json` and run it:
`"e2e": "node_modules/.bin/testcafe chrome e2e/"`

```
$ npm run e2e

> testcafe-mocha-example@1.0.0 e2e /Users/joeri/Documents/Developer/testcafe-mocha-example
> testcafe chrome e2e/

ERROR Cannot prepare tests due to an error.

Error: TypeScript compilation failed.
/testcafe-mocha-example/node_modules/@types/mocha/index.d.ts (42, 13): Subsequent variable declarations must have the same type.  Variable 'test' must be of type 'TestFn', but here has type 'ITestDefinition'.
```

So it is not about compiling the _rest_ of the code, but running the TestCafe files written in TypeScript. Did you manage to have those working with the example?

--- a/src/compiler/test-file/formats/typescript/compiler.js
+++ b/src/compiler/test-file/formats/typescript/compiler.js
@@ -21,7 +21,8 @@ export default class TypeScriptTestFileCompiler extends APIBasedTestFileCompiler
             lib:                     ['lib.es6.d.ts'],
             baseUrl:                 __dirname,
             paths:                   { testcafe: ['../../../../../ts-defs/index.d.ts'] },
-            suppressOutputPathCheck: true
+            suppressOutputPathCheck: true,
+            skipLibCheck:            true
         };
     }
 

